src/aere_sophia.erl

```diff
@@ -72,6 +72,8 @@ parse_body(I) ->
        end).
 parse_type(I) ->
     ?with_error_handle(aeso_parser:run_parser(aeso_parser:type(), I)).
+parse_typedef_variant(I) ->
+    ?with_error_handle(aeso_parser:run_parser(aeso_parser:typedef(variant), I)).
```

Comment:
Comment by radrow copied from an older PR:

Why not parse anything and make validation later?